ops/charm.py

```diff
@@ -369,11 +369,12 @@ class RelationEvent(HookEvent):
     """
 
     relation: 'Relation'
-    """The relation involved in this event."""
+    """The relation involved in this event.
+    """
```

Comment:
Style nit: for one-liner docstrings the quotes all go on the same line, but for multiline ones the end `"""` quote on a new line.